server/testdata/endpoints/04_couper.hcl

```diff
@@ -8,13 +8,14 @@ server "api" {
       proxy {
         backend {
           origin = "${request.headers.x-origin}"
-		  path   = "/resource"
+          path   = "/resource"
 
-		  oauth2 {
+          oauth2 {
             token_endpoint = "${request.headers.x-token-endpoint}/oauth2"
             client_id      = "user"
             client_secret  = "pass"
             grant_type     = "client_credentials"
+            retries        = 2
```

Comment:
Testcase for `0` is missing.

Reply:
And if the default of `1` is set and executed if the attribute is missing.

Reply:
Done